src/basilisp/core.lpy

```diff
@@ -1094,14 +1094,16 @@
   (operator/mod num div))
 
 (defn ^:inline quot
-  "Returns the quotient of ``num`` and ``div``\\."
+  "Returns the quotient of ``num`` and ``div``\\.
+
+  The result is rounded towards negative infinity."
   [num div]
   (basilisp.lang.runtime/quotient num div))
```

Comment:
```suggestion
  (operator/floordiv num div))
```

Please just delete `basilisp.lang.runtime.quotient` and convert to my suggestion above. This will allow us to directly optimize this code into the Python `num // div`.

Reply:
Done, I was wondering about this.